src/raw_node.rs

```diff
@@ -385,8 +385,8 @@ impl<T: Storage> RawNode<T> {
         false
     }
 
-    /// HasReady called when RawNode user need to check if any Ready pending.
-    /// Checking logic in this method should be consistent with Ready.containsUpdates().
+    /// Checks if any Ready is pending.
+    /// This method should be consistent with Ready.containsUpdates().
```

Comment:
Just for a hint, `containsUpdates()` seems not being implemented in this codebase.
It comes from https://github.com/etcd-io/etcd/blob/master/raft/node.go#L106

Reply:
Hmm maybe remove this line?